kairon/chat/handlers/channels/clients/whatsapp/cloud.py

```
@@ -170,3 +206,56 @@ def send_template_message(self, name: Text, to_phone_number, language_code: Text
+            url = f'{self.app}/{self.from_phone_number_id}/messages?{urlencode(self.auth_args)}'
+
+            async with RetryClient(raise_for_status=False, retry_options=retry_options) as client:
+                async with client.post(url, json=payload) as response:
+                    last_status_code = response.status
+                    if response.status == 200:
+                        resp = await response.json()
+                        return True, response.status, resp
+                    else:
+                        try:
+                            last_response = response.json()
+                        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)